docs/integrations-monitoring.mdx

```diff
@@ -0,0 +1,71 @@
+# Integrations monitoring
+
+Superface comes with out-of-the-box monitoring of integrations used by your App. With Superface project you will get insights on your App's API dependencies and their health status. In case of use-case failure, you will also receive an email notification with details about the failing API calls.
+
+## Set up integrations monitoring
+
+This guide assumes you have created a Node.js app with first integration. If you missed that, see the previous step: [Getting&nbsp;started](./getting-started.mdx)
+
+<div class="count-steps-h3">
+
+### Create new project
+
+Create a new project in [Superface dashboard](/dashboard).
+
+<!-- TODO: screenshot -->
+
+### Get your SDK token
+
+Copy the project's SDK token.
+
+### Use the SDK token with your Application
+
+In command line, set the token as `SUPERFACE_SDK_TOKEN` environment variable:
+
+```shell
+export SUPERFACE_SDK_TOKEN="<your token>"
+```
+
+Run your application again:
+
+```shell
+node index.js
+```
+
+### Check your dashboard
+
+Observe how are the capabilities used by your application.
+
+<!-- TODO: screenshot -->
```

Comment:
Yes, let's add a screenshot here, it will be also useful for quick demos

Reply:
Added screenshots for dashboard and email: https://pr-29-superface-docs.vercel.app/docs/integrations-monitoring